lib/gtfs/route_pattern.ex

```diff
@@ -6,6 +6,12 @@ defmodule Gtfs.RoutePattern do
 
   @type id :: String.t()
 
+  @doc """
+  A one-character id for disambiguating variants within a given route_id and direction_id
+  Either a digit or "_"
```

Comment:
Is a "_" something we actually want to (store and) display? That seems counter to my reading of the designs (but is ultimately a design/product question).

Reply:
[Decided not to display it](https://github.com/mbta/skate/pull/50#issuecomment-494501160)

I think it still makes sense to store it, because there's still a difference between "the default variant" and "no variants on this route", and it's a detail of the view that they're displayed the same, not the data model.

Reply:
👍 makes sense

Reply:
All routes have at least one variant/pattern, so I'm not sure there's actually a difference between those cases.

Reply:
Shuttles don't have route patterns.

Reply:
Indeed they don't! That's probably a bug though, since we do know what stops they're going to. I'll put in a ticket.